amplify.yml

```diff
@@ -27,6 +29,17 @@ backend:
         ##
         - export STRIPE_SECRET_KEY=${STRIPE_SECRET_KEY:-UNDEFINED}
         - make deploy
+        ##
+        # Export key deployed resources to System Manager Parameter Store
+        ##
+        - make export.parameter NAME="/${AWS_BRANCH}/stackName" VALUE=${STACK_NAME}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/id" VALUE=${COGNITO_USER_POOL_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/arn" VALUE=${COGNITO_USER_POOL_ARN}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/clientId" VALUE=${COGNITO_USER_POOL_CLIENT_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/api/id" VALUE=${GRAPHQL_API_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/api/url" VALUE=${GRAPHQL_URL}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/storage/flightTable" VALUE=${FLIGHT_TABLE_NAME}
```

Comment:
Following my first comment, these would be /service/amplify/storage/table/flight

And similar for below